ui/src/app/cohort-search/cohort-search/cohort-search.component.spec.tsx

```diff
@@ -41,7 +41,7 @@ describe('CohortSearch', () => {
   beforeEach(() => {
     currentWorkspaceStore.next(workspaceDataStub);
     registerApiClient(CohortBuilderApi, new CohortBuilderServiceStub());
-    serverConfigStore.set({config: {...defaultServerConfig, enableStandardSourceDomains: false}});
+    serverConfigStore.set({config: {...defaultServerConfig}});
```

Comment:
Should be able to remove this line completely along with the import statements above for `serverConfigStore` and `defaultServerConfig`